lib/bolt/pal.rb

```diff
@@ -29,15 +29,12 @@ def self.from_error(err)
         message = err.cause ? err.cause.message : err.message
 
         # Provide the location of an error if it came from a plan
-        details = if defined?(err.file) && err.file
-                    { file:   err.file,
-                      line:   err.line,
-                      column: err.pos }.compact
-                  else
-                    {}
-                  end
-
-        e = new(message, details)
+        details = {}
+        details[:file]    = err.file if defined?(err.file)
+        details[:line]    = err.line if defined?(err.line)
+        details[:column]  = err.pos if defined?(err.pos)
```

Comment:
```suggestion
        details[:file]   = err.file if defined?(err.file)
        details[:line]   = err.line if defined?(err.line)
        details[:column] = err.pos if defined?(err.pos)
```

Not sure how rubocop missed this... it usually doesn't like extra spaces around operators. :|